po/pl.po

```diff
@@ -696,149 +786,153 @@ msgstr "Skopiuj do schowka"
 msgid "Delete Taken Screenshot(s)"
 msgstr "Usuń wykonany(e) zrzut(y) ekranu"
 
-#: data/ui/image_sidebar.blp:253 data/ui/welcome_page.blp:86
+#: data/ui/image_sidebar.blp:253 data/ui/welcome_page.blp:84
 msgid "About Gradia"
 msgstr "O programie"
 
-#: data/ui/image_stack.blp:82
+#: data/ui/image_stack.blp:7 data/ui/welcome_page.blp:6
+msgid "Drop Image Here"
+msgstr "Upuść obraz tutaj"
+
+#: data/ui/image_stack.blp:97
 msgid "Reset Crop Selection"
 msgstr "Zresetuj zaznaczenie kadrowania"
 
-#: data/ui/image_stack.blp:144
+#: data/ui/image_stack.blp:124
+msgid "Confirm"
+msgstr "Potwierdź"
+
+#: data/ui/image_stack.blp:125
+msgid "Confirm Crop Selection"
+msgstr "Potwierdź zaznaczenie kadrowania"
+
+#: data/ui/image_stack.blp:226
 msgid "Clear All"
 msgstr "Wyczyść wszystko"
 
+#: data/ui/recent_picker.blp:21
+msgid "Screenshot Folder Not Found"
+msgstr "Nie odnaleziono katalogu zrzutów ekranu"
+
+#: data/ui/recent_picker.blp:28
+msgid "Please set it in the Preferences menu."
+msgstr "Ustaw w Preferencje."
+
 #: data/ui/text_entry_popover.blp:12
 msgid "Enter text…"
 msgstr "Wprowadź tekst…"
 
-#: data/ui/welcome_page.blp:25
+#: data/ui/welcome_page.blp:22
 msgid "Enhance an Image"
 msgstr "Ulepsz obraz"
 
-#: data/ui/welcome_page.blp:26
+#: data/ui/welcome_page.blp:23
 msgid "Drag and drop one here"
 msgstr "Przeciągnij i upuść tutaj"
 
-#: data/ui/welcome_page.blp:42
+#: data/ui/welcome_page.blp:39
 msgid "_Take a Screenshot…"
 msgstr "_Wykonaj zrzut ekranu…"
 
-#: data/ui/welcome_page.blp:54
+#: data/ui/welcome_page.blp:51
 msgid "_Open Image…"
 msgstr "_Otwórz obraz…"
 
-#: data/ui/preferences_window.blp:7 data/ui/preferences_window.blp:11
-msgid "Options"
-msgstr "Opcje"
+#: data/ui/preferences_window.blp:32
+msgid "Screenshots Folder"
+msgstr "Katalog zrzutów ekranu"
 
-#: data/ui/preferences_window.blp:15
-msgid "Recent Screenshots"
-msgstr "Ostatnie zrzuty ekranu"
-
-#: data/ui/preferences_window.blp:18
+#: data/ui/preferences_window.blp:35
 msgid "Select Folder"
 msgstr "Wybierz katalog"
 
-#: data/ui/preferences_window.blp:19
+#: data/ui/preferences_window.blp:36
 msgid "What folder to use for the home page"
 msgstr "Katalog domowy"
```

Comment:
Completely incorrect. Change to "Który folder użyć na stronie głównej".

Reply:
I checked it in app again and I agree with you, but it needs to be something else than "Katalog domowy". Maybe "Wybierz folder używany przez stronę startową"?